Bug/#538/seed example addition error

aevo98765 opened the pull request.

vishnoianil commented:
@aevo98765 Thanks for the PR and welcome back !!

Few review comments:
- This change needs to go in github knowledge form as well.
- The current delete buttons takes more real state compared to old one, wondering if we can think of something that doesn't require the form area to shirnk.
- Is this error specific to the local environment (when ran in npm dev mode)? because qa.ui.instructlab.ai doesn't throw any error when i add seed example to both skill and knowledge.

Existing form:
<img width="1117" alt="Screenshot 2025-02-03 at 10 15 00 PM" src="https://github.com/user-attachments/assets/23616692-eb93-49f4-b199-201a4d9e22fe" />

Change in this PR:
<img width="1072" alt="Screenshot 2025-02-03 at 10 14 37 PM" src="https://github.com/user-attachments/assets/c8211f90-04aa-428c-9b4c-8b9cd035b058" />

aevo98765 commented:
I have changed this so: 1) The delete button is only displayed when the accordion is closed. 2) Changed the layout proportions so that the delete button takes up less room.

aevo98765 commented:
Yeh it looks like when the project is built and started then this error does not occur. Happy to just have this error existing if the end user is never going to encounter it.

vishnoianil commented:
I think we can defer it till we hit it in prod. There are few new design changes are proposed by UX team as well, and that might help resolve this issue as well.

vishnoianil commented:
@aevo98765 Closing this PR. I believe the refactoring done across multiple PR addressed this issue.
